bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSTVSocket.java

```diff
@@ -332,6 +333,10 @@ private void sendMessage(JsonObject json) {
     @OnWebSocketMessage
     public void onMessage(String message) {
         logger.trace("Message [in]: {}", message);
+        if (this.listener == null) {
```

Comment:
Don’t like this change 
this class should be agnostic to OH types like thing handler and lifecycle

Reply:
This class is already linked to OH as you are updating the thing properties and the thing configuration when handling received messages (during the registering process). Of course, we have to avoid that when the thing handler is already disposed.
But we could moved this test before updating thing properties/configuration.

Reply:
therefore there is an interface which thinghandler implements

the behavior should not change whether or not somebody is whatching. reminds me of schrödinger’s cat